functional-test/src/test/groovy/org/rundeck/tests/functional/api/export_import/JobsExportSpec.groovy

```diff
@@ -0,0 +1,237 @@
+package org.rundeck.tests.functional.api.export_import
+
+import okhttp3.Headers
+import org.rundeck.util.annotations.APITest
+import org.rundeck.util.container.BaseContainer
+import spock.lang.Shared
+
+@APITest
+class JobsExportSpec extends BaseContainer {
+
+    @Shared String jobId
+
+    def setupSpec() {
+        startEnvironment()
+        setupProject()
+        def path = updateFile("job-template-common.xml")
```

Comment:
the method "updateFile" should receive a map with key and value to know which values to replace, instead of having X number of parameters, since we could have an base xml with a different structure

Reply:
Switch is made to use Map.